draft-ietf-openpgp-pqc.md

```diff
@@ -533,9 +459,7 @@ TBD                   | SLH-DSA-SHAKE-256s                 | MAY         | {{slh
 
 \[ Note: this section to be removed before publication \]
 
-Algorithms indicated as MAY are not assigned a codepoint in the current state of the draft
-since there are not enough private/experimental code points available
-to cover all newly introduced public-key algorithm identifiers.
+Algorithms indicated as MAY are not assigned a codepoint in the current state of the draft since there are not enough private/experimental code points available to cover all newly introduced public-key algorithm identifiers.
```

Comment:
@wussler

Actually, currently this statement still applies since we haven't yet split out the NIST/Brainpool composites. I will make an issue for the split where I note that we need to remove this statement then as well.

Reply:
I was referring to the SLH-DSA